src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -208,7 +207,7 @@ const ChartsHolder = (props: {
               {testPositiveData && (
                 <>
                   <ChartPositiveTestRate columnData={testPositiveData} />
-                  <Disclaimer metricName="positive test rate">
+                  <Disclaimer metricName={Metric.POSITIVE_TESTS}>
                     {POSITIVE_RATE_DISCLAIMER}
```

Comment:
Hrm...  This seems fine, but FWIW- It seems a bit weird that we are passing in _part_ of the disclaimer text and then the Disclaimer component optionally adds some more disclaimer text based on the metricName...

It feels like ideally we should either pass in _all_ of the text or have the Disclaimer component generate _all_ of the text (based on metricName).

Like this it's kinda' fragmented / confusing how the text is getting generated.

Reply:
hmm yeah this is true. added in a TODO

Reply:
Perfect! 👍